sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetFileFormat.scala

```diff
@@ -62,7 +63,14 @@ object GpuParquetFileFormat {
         s"compression codec ${parquetOptions.compressionCodecClassName} is not supported"))
 
     if (sqlConf.writeLegacyParquetFormat) {
-      meta.willNotWorkOnGpu(s"Spark legacy format is not supported")
+      meta.willNotWorkOnGpu("Spark legacy format is not supported")
+    }
+
+    if (!meta.conf.isParquetInt96WriteEnabled && sqlConf.parquetOutputTimestampType ==
+      ParquetOutputTimestampType.INT96) {
+      meta.willNotWorkOnGpu("Writing INT96 is disabled, if you want to enable it turn it on by " +
+        "setting the `spark.rapids.sql.format.parquet.writer.int96.enabled` to true. NOTE: check " +
```

Comment:
nit: we often use something like `s"setting the ${RapidsConf.ENABLE_PARQUET_INT96_WRITE.key} to true."` just to avoid any possible typos.

Reply:
Fixed